gateway/cli/src/lightning_commands.rs

```diff
@@ -93,6 +105,20 @@ impl LightningCommands {
                     .await?;
                 println!("{response}");
             }
+            Self::PayInvoice {
+                invoice,
+                max_delay,
+                max_fee_msats,
+            } => {
+                let response = create_client()
+                    .pay_invoice(ln_gateway::rpc::PayInvoicePayload {
+                        invoice,
+                        max_delay,
+                        max_fee: max_fee_msats,
+                    })
+                    .await?;
+                println!("{response}");
```

Comment:
Is there a reason why we're not using `print_response`?

Reply:
Yes, for types that implement `std::fmt::Display`, `print_response()` will wrap the value in quotations where as using `println!()` simply prints the value.